Wikipedia/Code/WMFSettingsViewController.m

```diff
@@ -653,7 +653,7 @@ - (void)accountViewControllerDidTapLogout:(WMFAccountViewController *_Nonnull)ac
 //}
 
 - (void)userDidTapProfile {
-    WMFProfileCoordinator *profileCoordinator = [WMFProfileCoordinator profileCoordinatorForSettingsProfileButtonWithNavigationController:self.navigationController theme:self.theme dataStore:self.dataStore logoutDelegate:self isExplore:YES];
+    WMFProfileCoordinator *profileCoordinator = [WMFProfileCoordinator profileCoordinatorForSettingsProfileButtonWithNavigationController:self.navigationController theme:self.theme dataStore:self.dataStore logoutDelegate:self sourcePage:0];
```

Comment:
(Optional) I've been trying to think of a way you could pass in something a bit more descriptive here. One thought is in ProfileCoordinator, you could add a method like this:

```
     @objc static var exploreOptOutSource: Source {
        return .exploreOptOut
    }
``` 

Then pass it in here like `logoutDelegate:self sourcePage:WMFProfileCoordinator.exploreOptOutSource];`

Still kind of icky. Alternatively you could move the source enum outside of ProfileCoordinator:

```
@objc
enum ProfileCoordinatorSource: Int {
    case exploreOptOut
    case explore
    case article
}

@objc(WMFProfileCoordinator)
final class ProfileCoordinator: NSObject, Coordinator, ProfileCoordinatorDelegate {
...
```

Then I think you will be able to pass in `ProfileCoordinatorSourceExploreOptOut` here instead of `0`.

Reply:
I liked the second one so that's what I went with :)